src/tools/WSEquipment.js

```diff
@@ -76,9 +76,10 @@ class WSEquipment {
         return WS._get(`/autocomplete/eqp/bin?code=${filter}&store=${store}`, config);
     }
 
-    autocompleteEquipmentCategory(filter, config = {}) {
+    autocompleteEquipmentCategory(filter, eqpClass, config = {}) {
         filter = encodeURIComponent(filter);
-        return WS._get(`/autocomplete/eqp/category/${filter}`, config);
+        eqpClass = eqpClass === null ? '' : '?class=' + encodeURIComponent(eqpClass);
```

Comment:
You can use the config instead for the query parameter.

See:
https://stackoverflow.com/questions/53501185/how-to-post-query-parameters-with-axios